src/main/java/duke/command/DeadlineCommand.java

```diff
@@ -0,0 +1,62 @@
+package duke.command;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+import duke.Storage;
+import duke.Ui;
+import duke.exceptions.InvalidFileException;
+import duke.exceptions.InvalidInputException;
+import duke.tasks.Deadline;
+import duke.tasks.Task;
+import duke.tasks.TaskList;
+
+/**
+ * Deadline Command class to execute command that adds a deadline to
+ * the TaskList.
+ */
+public class DeadlineCommand extends Command {
+
+    public static final String MESSAGE_SUCCESS = "Got it. I've added this task:\n";
+    public static final String MESSAGE_PARSE_ERROR = "Invalid date and time format.\n"
+            + "Please enter date and time in the format: yyyy-MM-dd HH:mm";
+
+    public DeadlineCommand(String input) {
+        super(input);
+    }
+
+    /**
+     * Execute a Deadline Command to add a Deadline into TaskList.
```

Comment:
it will be better to add an empty line between the javadoc description and @param